pydis_site/templates/home/index.html

```diff
@@ -19,21 +19,21 @@ <h1 class="is-size-1">Who are we?</h1>
         <div class="column is-half">
           <p>
             We're a large community focused around the Python programming language.
-            We believe anyone can learn programming, and are very dedicated to helping
-            novice developers take their first steps into the world of code. We also
+            We believe anyone can learn to code, and are very dedicated to helping
+            novice developers take their first steps into the world of programming. We also
             attract a lot of expert developers who are seeking friendships, collaborators,
-            or looking to hone their craft by teaching and getting involved in the community.
+            or to hone their craft by teaching and getting involved with the community.
 
             <br/><br/>
 
-            We organise regular community events like code jams, open source hackathons,
-            seasonal events and community challenges. Through our sponsorships and with
-            help from donations, we are even able to award prizes to the winners of our events.
+            We organise regular community events such as code jams, open-source hackathons,
+            seasonal events and community challenges. Through our sponsorships, and with
+            the help from donations, many of our events even have prizes to win!
 
             <br/><br/>
 
             You can find help with most Python-related problems in one of our help channels.
-            Our staff of nearly 50 dedicated expert Helpers are available around the clock
+            Our staff of over 50 dedicated expert Helpers, are available around the clock
```

Comment:
Should we really be calling ourselves experts here? Seems a bit presumptuous. Maybe it's just me who doesn't feel that way about myself 🤔

Reply:
I mean, I'm calling _you_ an expert more so than myself. Maybe that's why I'm comfortable with it. If anyone else finds this problematic I'd be happy to consider alternatives.

Reply:
> Expert: a person who is very knowledgeable about or skilful in a particular area.

It's not claiming anything we don't feel about our helpers already. I'm fine with it.